src/utilities/time.ts

```diff
@@ -662,7 +662,7 @@ export function getDoyTime(date: Date, includeMsecs = true): string {
   const { doy, hours, mins, msecs, secs, year } = getDoyTimeComponents(date);
   let doyTimestamp = `${year}-${doy}T${hours}:${mins}:${secs}`;
 
-  if (includeMsecs) {
+  if (includeMsecs && date.getMilliseconds() > 0) {
```

Comment:
Do you mind updating the function documentation example to reflect the change where if there are 0 ms, then it won't show the milliseconds?